docs/Systems/Tailwind/migration-to-v2.mdx

```diff
@@ -0,0 +1,25 @@
+import { Meta } from "@storybook/blocks"
+
+<Meta title="Systems/Tailwind/Migration to v2" />
+
+# Migration to v2
```

Comment:
Sorry for the basic question, but v2 of what? Not of Tailwind, surely? Of KAIO? It might be good to be explicit to avoid confusion.

Reply:
To try to avoid being verbose in the title, I've renamed the entire section to "Tailwind Preset" on our storybook to be more explicit. Hope that helps!